src/main/java/duke/tasks/Todo.java

```diff
@@ -0,0 +1,65 @@
+package duke.tasks;
+
+import java.util.Hashtable;
+import java.util.stream.Stream;
+
+import duke.exceptions.MissingInformationException;
+import duke.exceptions.MissingParameterException;
+import duke.utils.Parser;
+
+/**
+ * This class inplements the Todo task type for the bot.
+ *
+ * @author delishad21
+ */
+public class Todo extends Task {
+
+    private static String REQUIRED_PARAMS[] = {"description"};
```

Comment:
Any reason you use SCREAMING_SNAKE_CASE here? Typically, this case is reserved for constants (variables that are `final`). I noticed the same issue in several other classes too.